decidim-proposals/app/forms/decidim/proposals/admin/proposal_base_form.rb

```diff
@@ -36,12 +38,13 @@ class ProposalBaseForm < Decidim::Form
         delegate :categories, to: :current_component
 
         def map_model(model)
+          body = translated_attribute(model.body)
+          @suggested_hashtags = Decidim::ContentRenderers::HashtagRenderer.new(body).extra_hashtags.map(&:name).map(&:downcase)
```

Comment:
Another little question, here, I understand that this change implies that the @suggested_hashtags's variable value must always be available, not having yet defined a model.categorization right?

Reply:
That's correct!